src/carExample/Car.java

```diff
@@ -15,12 +16,12 @@
     private List<CarDoors> doors = new ArrayList<>();
 
     public Car(String productionDate) {
```

Comment:
why do you pass String productionDate change it to LocalDate as you've changed above